pkgs/applications/science/math/sage/sage.nix

```diff
@@ -61,6 +61,7 @@ stdenv.mkDerivation rec {
   };
 
   meta = with stdenv.lib; {
+    broken = true;
```

Comment:
Please do not mark Softmaker Office or FreeOffice broken. Hash mismatches occur every time upstream updates SoftMaker/FreeOffice (because they replace old versions), but this is usually fixed in a few days at most.

Reply:
Also, SoftMaker/FreeOffice currently seem to build fine?

Reply:
Will do.